www/js/search.js

```diff
@@ -22,7 +22,7 @@ const allowedKeys = [
 const sessionList = [];
 const sessionStatesList = {};
 const currentShabad = [];
-let currentShabadId = 0;
+let currentShabadState = 0;
```

Comment:
maybe initialize this `{id: null, type: null}`? 

that way you won't get any "can't not access prop of" issues